app/javascript/consent-form.vue

```diff
@@ -53,7 +53,7 @@
               >
                 <div class="steps__general text-center py-2 mb-30"
                   v-if="qstionGrp.header">
-                  <b h-html="qstionGrp.header"/>
+                  <b v-html="qstionGrp.header"/>
```

Comment:
This was broken in https://github.com/Australian-Genomics/CTRL/pull/83. https://github.com/Australian-Genomics/CTRL/issues/17 would have caught this kind of issue.

Reply:
Ah so true (about #17 ) When reviewing the PR after figuring out that's a Vue tag and what it does and didn't connect it with the intent 🤦‍♂️